resources/measures/upgrade_hvac_economizer/tests/hvac_economizer_test.rb

```diff
@@ -171,11 +169,156 @@ def apply_measure_and_run(test_name, measure, argument_map, osm_path, epw_path,
     return result
   end
 
+  def get_design_oa_flow_rates(model)
+    hash_oa_design_rates = {}
+    # get OA design rates prior to measure implementation
+    model.getSizingSystems.each do |sizing_system|
+
+      # get related airloophvac
+      air_loop_hvac = sizing_system.airLoopHVAC
+      name_air_loop_hvac = air_loop_hvac.name.to_s
+
+      # get design OA flow rate
+      oa_design_rate = nil
+      if sizing_system.autosizedDesignOutdoorAirFlowRate.is_initialized
+        oa_design_rate = sizing_system.autosizedDesignOutdoorAirFlowRate.get
+      elsif sizing_system.designOutdoorAirFlowRate.is_initialized
+        oa_design_rate = sizing_system.designOutdoorAirFlowRate.get
+      else
+        raise 'no design OA flow rate found'
+      end
+      puts("### DEBUGGING: original | name_air_loop_hvac = #{name_air_loop_hvac} | oa_design_rate = #{oa_design_rate}")
+
+      # add key (airloop name) and value (design OA rate)
+      hash_oa_design_rates[name_air_loop_hvac] = oa_design_rate.round(6)
+
+    end
+
+    return hash_oa_design_rates
+  end
+
+  def economizer_available(model)
+    economizer_availability = []
+    model.getAirLoopHVACs.each do |air_loop_hvac|
+      # get airLoopHVACOutdoorAirSystem
+      oa_sys = air_loop_hvac.airLoopHVACOutdoorAirSystem
+      if oa_sys.is_initialized
+        oa_sys = oa_sys.get
+      else
+        OpenStudio.logFree(OpenStudio::Error, 'openstudio.prototype.Model', "#{air_loop.name} is required to have an economizer, but it has no OA system.")
+        next
+      end
+      # get controller:outdoorair
+      oa_control = oa_sys.getControllerOutdoorAir
+      # change/check settings: control type
+      if oa_control.getEconomizerControlType != 'NoEconomizer'
+        economizer_availability << true
+      else
+        economizer_availability << false
+      end
+    end
+    return economizer_availability
+  end
+
+  def models_to_test_design_oa_rates
```

Comment:
We should update these models with 3.7 models. I have some on the origin/ccaradon/euss_hprtu_hr branch, but others are being added to the origin branch soon.